comment-updater-core/src/main/kotlin/org/jetbrains/research/commentupdater/processors/MethodChangesExtractor.kt

```diff
@@ -27,7 +28,7 @@ object MethodChangesExtractor {
                     override fun visitDocComment(comment: PsiDocComment?) {
                         if (comment != null) {
                             if (comment.owner is PsiMethod) {
```

Comment:
can we combine these two checks?
```
if (comment != null and comment.owner is PsiMethod) {
...
}
```

I haven't written in Kotlin for a couple of years, so I'm not sure about the syntax